Server/Classes/Services/ResourceLocationOwnerService.sqx

```diff
@@ -59,10 +59,15 @@ namespace Intrusion.Server
 		// Returns (Side): The dominant side. civilian if sides are equally dominant.
 		private method Side CheckDominantSide("_markerName" as String)
 		{
-			private ["_side" as Side, "_points" as Scalar, "_bestSide" as Side, "_bestSidesPoints" as Scalar];
+			private ["_side" as Side, "_points" as Scalar, "_bestSide" as Side, "_bestSidesPoints" as Scalar, "_excludedUnits" as Array];
 			
 			_bestSide = civilian;
 			_bestSidesPoints = 0;
+			_excludedUnits = [];
+			{
+				// allUnitsUAV is returning all UAV-vehicles and not the units inside.
+				_excludedUnits = _excludedUnits + crew _x;
```

Comment:
But... This seems to exclude the crew of UAVs in all cases. What if the UAV is up flying somewhere far away and the controller guy walks in to capture a flag? It looks like he will not be able...